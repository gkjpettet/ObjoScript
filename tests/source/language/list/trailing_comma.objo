#  Expect parsing error due to trailing comma.
var list = ["a", "b",]