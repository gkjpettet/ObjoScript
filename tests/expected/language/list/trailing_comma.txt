Expected an expression. Instead got `EOL`.